gap/curl.gd

```diff
@@ -75,7 +75,7 @@ DeclareGlobalFunction("DownloadURL");
 #!   <Ref Func="CurlRequest"/>.
 #!
 #! @BeginExample
-#! gap> r := PostToURL("www.httpbin.org/post", "animal=tiger");;
+#! gap> r := PostToURL("httpbun.com/post", "animal=tiger");;
```

Comment:
I am confused: this changes from `www.httpbin.org` (with an `i`) to `httpbun.com` (with a `u` and also `.com` not `.org`) which is not quite what the PR description suggests?

Reply:
In other words: Maybe "Use httpbun.com insteadof `www.httpbin.org" would a more appropriate PR title? Or am I missing something?